.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+app.log
```

It's better to ignore all folder with logs
```suggestion
logs/*
```